modules/contact-form/grunion-contact-form.php

```diff
@@ -2553,6 +2565,16 @@ function render() {
 		$field_required    = $this->get_attribute( 'required' );
 		$placeholder       = $this->get_attribute( 'placeholder' );
 		$class             = $this->get_attribute( 'class' );
+		
+		/**
```

Comment:
Since the block above has already been documented, this can just be 

`/** This filter is already documented in modules/contact-form/grunion-contact-form.php */`